aten/src/ATen/SparseCsrTensorImpl.cpp

```diff
@@ -44,20 +57,24 @@ SparseCsrTensorImpl::SparseCsrTensorImpl(
               at::initialTensorOptions()
                   .device(SparseCsrTensorSetToDeviceType(key_set))
                   .dtype(data_type)) // values
+          ,
+          layout
       ) {}
 
 SparseCsrTensorImpl::SparseCsrTensorImpl(
     at::DispatchKeySet key_set,
     const caffe2::TypeMeta data_type,
     at::Tensor crow_indices,
     at::Tensor col_indices,
-    at::Tensor values)
+    at::Tensor values,
+    Layout layout)
     : TensorImpl(key_set, data_type, values.device()),
       crow_indices_(std::move(crow_indices)),
       col_indices_(std::move(col_indices)),
-      values_(std::move(values)) {
+      values_(std::move(values)),
+      layout_(layout) {
   // https://pytorch.org/blog/pytorch-feature-classification-changes/#beta
-  TORCH_WARN_ONCE("Sparse CSR tensor support is in beta state.");
+  TORCH_WARN_ONCE("Sparse ", SparseCsrTensorLayoutToSTRING(layout_), " tensor support is in beta state.");
```

Comment:
We can also extend this by telling the user to consider opening a feature request if there is missing functionality with a link to the page: https://github.com/pytorch/pytorch/issues

Reply:
Extended the warning message with "If you miss a functionality in the sparse tensor support, please submit a feature request to https://github.com/pytorch/pytorch/issues."

Reply:
Resolved via https://github.com/pytorch/pytorch/pull/75594